### PR TITLE
[repository overview] fix: overflow occurring on the repository overview page

### LINKED ISSUE #3807: Fix overflow occurring on the Repository Overview page
### Description

Currently, there is an overflow occurring on the [Repository overview page](https://layer5.io/community/handbook/repository-overview)

### Expected Behavior

Fix the overflow 

### Screenshots

rep-overview

### Environment:
- Host OS: 
- Browser: 

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by Aditya-A-G**
**Description**

This PR fixes #3807 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [&#x2714; ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 4800a3be5c42fe9d3f6f37934c923a31dc1a7e86 at: https://63fd598cccddc826b77892d9--layer5.netlify.app

**Review by tinniaru3005 — CHANGES_REQUESTED** [review]
Hey, @Aditya-A-G Great work. Just a small change: 
- Remove part 1 as marked in the image
- Mobe the heading to the top
rep-overview

**Review by leecalcote — COMMENTED** [review]
Thanks for this, @Aditya-A-G.

Gosh, does all of this need to change?

**Comment by Aditya-A-G** [comment]
Thanks for reviewing the pull request, @leecalcote  I made the changes proposed because there were inconsistent spacing issues across multiple pages of the community handbook. If you have any specific concerns or suggestions for improvement, please let me know and I will be happy to address them. Also are you good with the proposed suggestions by @tinniaru3005

**Comment by Aditya-A-G** [comment]
Additionally, I made some changes to the styles to improve readability by moving them around in the file

File: src/sections/Community/Handbook/Handbook.style.js [hunk 1]
```diff
@@ -26,25 +35,19 @@ export const HandbookWrapper = styled.div`
...
-        margin-top: -7rem;
-      }
-      h3{
-        padding-top: 7rem;
-        margin-top: -7rem;
-      }
-      margin-top: -36rem;
-      margin-left: 20rem;
-      display: flex;
-    }
```
Comment (Nikhil-Ladha) [inline src/sections/Community/Handbook/Handbook.style.js:46]: Apart from these lines there are no actual changes in this PR. Please revert the other changes and limit the changes only to those that are needed.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]

**Comment by Nikhil-Ladha** [comment]
Yea, please address the feeedback given by @tinniaru3005 .

> Additionally, I made some changes to the styles to improve readability by moving them around in the file

No neeed for such changes, please revert them.

**Comment by l5io** [comment]
🚀 Preview for commit 50e4022c0c23f3d8dd4fa56851ba64f0b9f04577 at: https://6401f03906ae110f5b42bbb8--layer5.netlify.app

**Comment by Nikhil-Ladha** [comment]
Please follow the steps mentioned [here](https://github.com/layer5io/layer5/pull/3834/checks?check_run_id=11742855318) to fix the DCO check.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Please disable any prettify plugin that you have and limit the changeset to only the required changes.

**Comment by l5io** [comment]
🚀 Preview for commit 31c9e2974045e52ac7e10ead6f91c3cacea864b7 at: https://640210e08bf47f2f6ac60dd9--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Review by tinniaru3005 — APPROVED** [review]
LGTM :)

**Pull request merged**